app/screens/Home/HomeView.js

```diff
@@ -1,76 +1,63 @@
 import React from 'react';
-import { View, Text, Image, FlatList } from 'react-native';
-import styles from './styles';
-import menuPic from '../../assets/img/menu.png';
-import searchPic from '../../assets/img/search.png';
-import Places from '../../components/places/Places';
-import boudha from '../../assets/img/boudha.jpeg';
-import patan from '../../assets/img/patan.jpeg';
-import basantapur from '../../assets/img/basantapur.jpeg';
-import chitwan from '../../assets/img/chitwan.jpeg';
+import { View, FlatList, TouchableOpacity } from 'react-native';
 
-export default class HomeView extends React.Component {
+import Places from '../../components/Places/Places';
 
-  placePressed = (place) => {
-    this.props.navigation.navigate('Info', {place: place});
-  }
+import data from './data';
 
-  render() {
-    return (
-      <View style={{flex:1}}>
-        <View style={styles.header}>
+import styles from './styles';
+
+import {Icon} from 'react-native-elements';
 
-          <View style={styles.menuBox}>
-            <Image style={styles.menuIcon} source={menuPic} />
-          </View>
 
-          <View style={styles.titleBox}>
-            <Text style={styles.title}>Discover</Text>
-          </View>
 
-          <View style={styles.searchBox}>
-            <Image style={styles.menuIcon} source={searchPic} />
-          </View>
+export default class HomeView extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      places: data || this.fetchData()
```

Comment:
dont fetchdata/initialize state on constructor. Better do it in componentDidMount()